legi/tar2sqlite.py

```diff
@@ -501,10 +502,10 @@ def main():
 
     # Look for new archives in the given directory
     print("> last_update is", last_update)
-    archive_re = re.compile(r'(.+_)?legi(?P<global>_global)?_(?P<date>[0-9]{8}-[0-9]{6})\..+')
+    archive_re = re.compile(r'(.+_)?(legi|LEGI)(?P<global>_global)?_(?P<date>[0-9]{8}-[0-9]{6})\..+')
     skipped = 0
     files = sorted(os.listdir(args.directory))
-    for archive_name in fnmatch.filter(files, '*legi_*.tar.*'):
+    for archive_name in chain(fnmatch.filter(files, '*legi_*.tar.*'), fnmatch.filter(files, '*LEGI_*.tar.*')):
```

Comment:
This change introduces a theoretical risk of processing the archives in the wrong order.

Reply:
Something like this should work

```python
    files = sorted([filename for filename in os.listdir(args.directory) if fnmatch.fnmatch(filename, '*LEGI_*.tar.*') or 
fnmatch.fnmatch(filename, '*legi_*.tar.*')], key=lambda s: s.lower())
    for archive_name in files:
        ...
```
but using the date from archive_re could be also an alternative

Reply:
I just pushed something like @nheir proposed, we are now exactly in the previous situation.
In another PR/commit, perhaps the mechanism to be improved by sorting by dates instead of relying on filenames (we are lucky that "freemium" is before "legi").